bin/validate-json

```diff
@@ -6,32 +6,18 @@
  * @author Christian Weiske <christian.weiske@netresearch.de>
  */
 
-/**
- * Dead simple autoloader
- *
- * @param string $className Name of class to load
- *
- * @return void
- */
-function __autoload($className)
-{
-    $className = ltrim($className, '\\');
-    $fileName  = '';
-    if ($lastNsPos = strrpos($className, '\\')) {
-        $namespace = substr($className, 0, $lastNsPos);
-        $className = substr($className, $lastNsPos + 1);
-        $fileName  = str_replace('\\', DIRECTORY_SEPARATOR, $namespace) . DIRECTORY_SEPARATOR;
-    }
-    $fileName .= str_replace('_', DIRECTORY_SEPARATOR, $className) . '.php';
-    if (stream_resolve_include_path($fileName)) {
-        require_once $fileName;
-    }
+// support running this tool from git checkout
+if (is_dir(__DIR__ . '/../vendor')) {
```

Comment:
I edited the comment to refer to directory separator; I didn't realize my phrasing collided with that other constant at the time I wrote it.

Screwing with relative paths in strings to wire things up always seems dirty to me, especially with directory separators differing between platforms.

An example of what I am suggesting:

```shell
$ cd /path/to/json-schema/bin
$ php -a
Interactive shell

php > echo __DIR__;
/path/to/json-schema/bin
php > echo dirname(__DIR__);
/path/to/json-schema
php > echo dirname(__DIR__) . DIRECTORY_SEPARATOR . 'vendor';
/path/to/json-schema/vendor
```

What do you think?